doc/sphinx-guides/source/installation/oauth2.rst

```diff
@@ -1,4 +1,4 @@
-OAuth Login: ORCID, GitHub, Google
+OAuth Login: ORCID, Microsoft Azure Active Directory, GitHub, Google
```

Comment:
Sorry for the noise, but when I see this now, it comes to my mind that this title is going to be included in the TOC and sidebar of the guide. It will be a very long title, getting wrapped. When more providers are added, that list will grow even longer. Is it time to cut off and just name it "OAuth2-based Login" or similar? (Or is there an option to create a short title for TOCs?)

Reply:
@poikilotherm for a long time I've thought of creating an issue at https://github.com/IQSS/dataverse-ansible/issues called "build guides" or something but I also noticed that you created #6188 and I wonder if that would make things easier. What I'm thinking is that our spin up scripts are now always being executed (thanks to all the effort by @donsizemore 🎉 ) and perhaps we can teach these scripts (by way of dataverse-ansible) to build the guides too so they can be reviewed along with pull requests. Some pull requests **only** touch the guides. 😄

Reply:
I shortened this, more scalable. https://github.com/IQSS/dataverse/pull/6192/commits/4931d6afaf2d58d4bca0b4255655c9d43076a882